specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/expressRoutePort.json

```diff
@@ -542,6 +542,65 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/expressRoutePorts/{expressRoutePortName}/generateLoa": {
+      "post": {
+        "tags": [
+          "ExpressRoutePorts"
+        ],
+        "operationId": "ExpressRoutePorts_GenerateLOA",
+        "description": "Generate LOA for the requested ExpressRoutePort resource.",
```

Comment:
>LOA [](start = 33, length = 3)

Can you expand the LOA acronym in all the descriptions?

Reply:
Letter of authorization